bugfix/assignee_type_deprecation (#75)
fivetran/dbt_intercom: CHANGELOG.md

# dbt_intercom v1.5.0

[PR #75](https://github.com/fivetran/dbt_intercom/pull/75) includes the following updates:

## Schema/Data Change
**3 total changes • 2 possible breaking change**

| Data Model(s) | Change type | Old | New | Notes |
| ------------- | ----------- | ----| --- | ----- |

> | `intercom__conversation_enhanced`<br>`intercom__conversation_metrics` | New field | | `conversation_assignee_id` | Exposes the conversation assignee ID (the ID of the user assigned to the conversation) from the staging layer for reference and joins. If you persist `assignee_id` via the `intercom__conversation_history_pass_through_columns` variable, ensure its alias does not match `conversation_assignee_id` to avoid duplicate column errors.  |

Reviewer: The wording here is a bit clunky and confusing. I'd suggest something like: The ID of the user assigned to the Intercom conversation. This column is an alias of `assignee_id` from `stg_intercom__conversation_history`. If you also persist `assignee_id` using the `intercom__conversation_history_pass_through_columns` variable, ensure it uses a different alias to avoid duplicate column errors.

Author: Thanks, will proceed with the below re-wording suggestion.
```suggestion
| `intercom__conversation_enhanced`<br>`intercom__conversation_metrics` | New field | | `conversation_assignee_id` | The ID of the user assigned to the conversation. This column is an alias of `assignee_id` from `stg_intercom__conversation_history` model. If you also persist `assignee_id` using the `intercom__conversation_history_pass_through_columns` variable, ensure it uses a different alias to avoid duplicate column errors.  |
```
